service/src/main/java/bio/terra/workspace/service/iam/SamService.java

```diff
@@ -629,6 +629,29 @@ public String syncWorkspacePolicy(
     return group;
   }
 
+  /**
+   * Retrieve the email of a sync'd workspace policy using the WSM SA credentials. This is used
+   * during controlled resource create and the caller may not have access to the workspace policies.
+   *
+   * @param workspaceId workspace to use
+   * @param role workspace role to lookup
+   * @return email of the sync'd policy group
+   * @throws InterruptedException on shutdown during retry wait
+   */
+  public String getWorkspacePolicyAsWsm(UUID workspaceId, WsmIamRole role)
+      throws InterruptedException {
+    GoogleApi googleApi = samGoogleApi(getWsmServiceAccountToken());
```

Comment:
That is weird. I assumed it would require `policy_read` and the caller probably wouldn't have that.
I'll change it to use passed credentials, but it seems wrong to me that anyone can see the sync state.

Reply:
I posted the question about anyone being able to read sync state to the CI team trello board.